test/l2/dai.ts

```diff
@@ -84,10 +84,21 @@ describe('Dai', () => {
         ).to.be.revertedWith('')
       })
 
+      it('should not transfer to dai address', async () => {
+        await expect(dai.connect(signers.user1).transfer(dai.address, 1)).to.be.revertedWith('')
+        await expect(dai.connect(signers.user1).transferFrom(signers.user1.address, dai.address, 1)).to.be.revertedWith(
```

Comment:
just a note, `revertedWith` expects a message error to be passed as parameter. if that's not the case `reverted` is the preferred way.